continuous-integration/python3.sh

```diff
@@ -3,3 +3,6 @@
 # This script runs a passing subset of tests under Python 3.
 
 python setup.py install
+python -c "import sunpy.data"
+python -c "import sunpy.data; sunpy.data.download_sample_data()"
+python -c "import sunpy.data.sample"
```

Comment:
just to make is cleaner and easier to see which one was broken. This script is only temporary until the whole test suite works under 3.x